package.json

```diff
@@ -68,7 +68,7 @@
   },
   "author": "engineering@strongloop.com",
   "license": {
-    "name": "Dual Artistic/StrongLoop",
+    "name": "Artistic-2.0 OR DocumentRef-LICENSE.md:LicenseRef-StrongLoop",
```

Comment:
I was following the grammar used by the spdx module, didn't even notice that npm's docs now cover this case.